scripts/cmake/ElektraCache.cmake

```diff
@@ -354,6 +354,20 @@ if (CMAKE_SKIP_BUILD_RPATH AND BUILD_TESTING)
 	message (WARNING "You specified to remove RPATH, but did not disable tests. Please read doc/COMPILE.md#RPATH")
 endif ()
 
+#
+# Packaging options
+#
+
+set (
+	CPACK_DEBIAN_PACKAGE_RELEASE
```

Comment:
Why not only one variable CPACK_PACKAGE_RELEASE?

Reply:
CPACK_DEBIAN_PACKAGE_RELEASE and CPACK_RPM_PACKAGE_RELEASE are both predefined variables of CPack, but I agree, caching only CPACK_PACKAGE_RELEASE and setting CPACK_DEBIAN_PACKAGE_RELEASE and CPACK_RPM_PACKAGE_RELEASE to CPACK_PACKAGE_RELEASE would be better.

Reply:
If they are already predefined we do not need to set them as cache variable anyway?

Reply:
I agree, but I would at least define CPACK_PACKAGE_RELEASE as cache variable with a default value. This is especially useful for our release pipeline where the package revision of each package can be changed with the parameterized build.

Reply:
Cache variables are afaik only needed if you want it to be visible in the cmake GUIs. I do not think this is needed in this situation.

With something like:
```
if (CPACK_RPM_PACKAGE_RELEASE) set (CPACK_RPM_PACKAGE_RELEASE ...)
```
you should be able to set a default if the variable is not set from outside.